sdk/core/src/options.rs

```diff
@@ -126,33 +132,36 @@ impl RetryOptions {
         mode: RetryMode => mode,
         delay: Duration => delay,
         max_retries: u32 => max_retries,
-        max_delay: Duration => max_delay,
+        max_elapsed: Duration => max_elapsed,
     }
 }
 
 impl Default for RetryOptions {
     fn default() -> Self {
         RetryOptions {
             mode: RetryMode::default(),
-            delay: Duration::from_millis(800),
-            max_retries: 3,
-            max_delay: Duration::from_secs(60),
+            delay: Duration::from_millis(200),
+            max_retries: 10,
+            max_elapsed: Duration::from_secs(60),
+            max_delay: Duration::from_secs(30),
```

Comment:
**Suggestion:** It might be worth trying to align the defaults, i.e. so that you aren't setting `max_retries` to a value that is unattainable due to `max_elapsed`. Otherwise it can be a bit misleading wrt to what behavior you would expect from the default. I guess this will be a bit hard because it differs by policy, so maybe a doc comment would also be enough?

Reply:
I added a document explaining this.